AmplifyPlugins/API/AWSAPICategoryPlugin/Configuration/AWSAPICategoryPluginConfiguration+InterceptorsConfig.swift

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import Foundation
+import AWSPluginsCore
+import AWSCore
+
+public extension AWSAPICategoryPluginConfiguration {
+    struct EndpointInterceptorsConfig {
```

Comment:
this struct is for holding an array of interceptors for a single apiEndpoint and i think you had mentioned that you wanted to move away from having interceptors in the configuration class, can we move this out out the `AWSAPICategoryPluginConfiguration` namespace and just call it `APIEndpointInterceptors`?

Reply:
Yes, let's rename it `AWSAPIEndpointInterceptors`